src/containers/other/NFTView/NFTView.module.css

```diff
@@ -46,11 +46,9 @@
 }
 
 .Image {
-	height: 422px;
-	min-height: 422px;
+	height: auto;
 	max-height: 422px;
 	cursor: pointer;
-	background: rgba(0, 0, 0, 0.3);
 	border-radius: inherit;
```

Comment:
We shouldn't have a border radius here - [relevant Figma frame](https://www.figma.com/file/0zFJaRNODss8lxwRygUh24/UI-Wilder-Market?node-id=5959%3A118358)